mediapipe/tasks/ios/MediaPipeTasksText.podspec.template

```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
 
   s.module_name = 'MediaPipeTasksText'
   s.static_framework = true
-  s.dependency 'MediaPipeTasksCommon'
+  s.dependency 'MediaPipeTasksCommon', '${MPP_TASKS_COMMON_VERSION}'
```

Comment:
Fixing `MPPTasksCommon` to a particular version. In case of changes in `MediaPipeTasksCommon in future versions, this will prevent any issues when installing older versions of text and vision pods.